README.md

```diff
@@ -105,13 +105,15 @@ about what it does.
 package tarantool
 
 import (
+	"context"
 	"fmt"
 	"github.com/tarantool/go-tarantool/v2"
 )
 
 func main() {
 	opts := tarantool.Opts{User: "guest"}
-	conn, err := tarantool.Connect("127.0.0.1:3301", opts)
+	ctx := context.Background()
```

Comment:
Let's use a context with timeout in the example.

Reply:
Updated. All tests now use either a context with timeout or a context with cancel.